src/components/LocationListener.tsx

```diff
@@ -70,15 +70,16 @@ class LocationListener extends Component<Props> {
   }
   onLocation = (location: Location) => {
     const {
-      coords: {latitude, longitude},
+      coords: {latitude, longitude, speed},
       timestamp,
     } = location
-
-    this.props.locationStorage.recordLocation({
-      latitude: latitude,
-      longitude: longitude,
-      when: timestamp,
-    })
+    if (speed && speed <= 10 * 2.23694 || !!speed == false) {
```

Comment:
two things: can you extract the magic number into a constant and `|| !!speed == false` seems really weird to me. can you write out in a comment  exactly what this predicate is doing?